### DISCUSSION TITLE
TreeNode 组件在拖动到顶部与底部的时候，如何扩大顶部与底部的占位符，不然目前的拖动在最底部或最顶部不是很友好，体验很差

### DISCUSSION THREAD

**Discussion by gaecoli** [ant-design/ant-design#35748] [General]
### What problem does this feature solve?

解决组件Tree组件拖动时候的体验
<img width="537" alt="image" src="https://user-images.githubusercontent.com/46274612/169979720-dcfda016-5f35-4fab-a890-87d018d21b1f.png">

最底部可移动区域太小了

### What does the proposed API look like?

是否可以将这一区域的gap增加，可以放置一个空行来放置拖动的位置，感谢

**Comment by afc163** [comment]
Tree 内部用 style 属性加个 padding bottom 是否可行呢？

**Reply by gaecoli** [reply]
> Tree 内部用 style 属性加个 padding bottom 是否可行呢？

已经解决了，感谢你！

**Comment by gaecoli** [comment]
> Tree 内部用 style 属性加个 padding bottom 是否可行呢？

感谢回复，我尝试一下！
